### PR TITLE
Update documentation based on new operator design

### LINKED ISSUE #114: Update documentation based on the new operator design
Part of the CodeFlare operator re-design, as described in https://github.com/project-codeflare/adr/pull/9.

Accommodate documentation, according to the new design of the operator, e.g., removing reference to the remove InstaScale and MCAD resources and deployments.

### PR DESCRIPTION
**Pull request by astefanutti**
Closes #114.

### CODE REVIEW

File: Quick-Start-ODH-V2.md [hunk 1]
```diff
@@ -76,12 +73,36 @@ At this point you should be able to go to your notebook spawner page and select
...
-If you are using an Openshift Dedicated or ROSA Cluster you will need to create a secret in the opendatahub namespace containing your ocm token. You can find your token [here](https://console.redhat.com/openshift/token). Navigate to Workloads -> secrets in the Openshift Console. Click Create and choose a key/value secret. Secret name: instascale-ocm-secret, Key: token, Value: < ocm token > and click create.
+
+If you want to enable cluster auto-scaling with InstaScale on Openshift Dedicated or ROSA, you will need to create a secret, containing your OCM token.
+You can find your token [here](https://console.redhat.com/openshift/token).
+Navigate to Workloads -> Secrets in the Openshift Console.
+Click Create and choose a key/value secret:
 
 <img src="images/instascale-ocm-secret.png" width="80%" height="80%">
 
+Then you'll have to edit the CodeFlare operator ConfigMap, with:
```
Comment (anishasthana) [inline Quick-Start-ODH-V2.md:85]: Should we add a note saying that the operator needs to be restarted after the CM is updated?

Comment (astefanutti) [reply Quick-Start-ODH-V2.md:85]: Good point, thanks. Added operator Pod restart as additional step.

**Review by anishasthana — APPROVED** [review]
/lgtm

**Pull request merged**